src/main/java/onlinebookstore/service/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package onlinebookstore.service.impl;
+
+import java.util.List;
+import onlinebookstore.entity.Book;
+import onlinebookstore.repository.BookRepository;
+import onlinebookstore.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
```suggestion
@Service
```
Use annotation Service for classes  with business logic